apps/dapp/src/components/Pages/Core/DappPages/SpiceBazaar/Earn/StakeTemple/Delegate.tsx

```diff
@@ -0,0 +1,493 @@
+                    checked={
+                      !isDelegatingToSelf ||
+                      currentDelegate?.toLowerCase() ===
+                        walletAddress?.toLowerCase()
+                    }
+                    onChange={() => setIsDelegatingToSelf(false)}
+                  />
+                  <RadioLabel htmlFor="another">Another address</RadioLabel>
+                </RadioOption>
+                {!isDelegatingToSelf && (
+                  <Input
+                    value={inputValue}
+                    handleChange={handleInputChange}
+                    placeholder={`${formatAddress(walletAddress || '')}`}
+                    width="100%"
+                  />
+                )}
+              </AnotherAddressContainer>
+            </RadioGroup>
+          </DelegateContainer>
+          <StakedContainer>
+            <StakedAmountContainer>
+              <StakeTempleIcon />
+              <Staked>
+                {formatNumberWithCommas(stakePageMetrics.data.yourStake)} STAKED
+                TEMPLE
+              </Staked>
+            </StakedAmountContainer>
+            <StakedTextBottom>
+              will be delegated to{' '}
+              {!isDelegatingToSelf && inputValue
+                ? formatAddress(inputValue)
+                : formatAddress(walletAddress || '')}
+            </StakedTextBottom>
+          </StakedContainer>
+          <TradeButton
+            style={{
+              whiteSpace: 'nowrap',
+              marginTop: '0px',
+              alignSelf: 'center',
+            }}
+            onClick={handleDelegateClick}
+            loading={isDelegating || isAnyDataLoading}
+            disabled={(() => {
+              if (!walletAddress) return true;
+              if (!isDelegatingToSelf && !inputValue) return true;
+
+              const selectedDelegate = getSelectedDelegateAddress();
+              if (
+                currentDelegate &&
+                selectedDelegate &&
+                selectedDelegate.toLowerCase() === currentDelegate.toLowerCase()
+              ) {
+                return true;
+              }
+
+              return false;
+            })()}
+          >
+            {currentDelegate ? 'CHANGE DELEGATE' : 'DELEGATE'}
+          </TradeButton>
+        </>
+      )}
+      {stakePageMetrics.data.yourStake === 0 && (
+        <>
+          <StakedTempleContainer>
+            <StakeTempleIcon />
+            <StakedTempleText>
+              <StakedAmount>
+                {formatNumberWithCommas(stakePageMetrics.data.yourStake)} STAKED
+                TEMPLE
+              </StakedAmount>
+              <StakedStatusText>
+                YOU CURRENTLY HAVE NO STAKED TEMPLE
+              </StakedStatusText>
+            </StakedTempleText>
+          </StakedTempleContainer>
+          <Padding>
+            <Line></Line>
+            <NoStakedTempleText>
+              Stake TEMPLE in the ‘Stake’ tab and come back here to delegate
```

Comment:
nit: would it help if there is a link to the tab here?

Reply:
https://github.com/TempleDAO/temple/pull/1197/commits/f2de04c1b629f5bd801bcade6a7abcf486890883